lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-colors';
+
+.nx-binary-donut-chart {
+  width: 30px;
+  height: 30px;
+
+  circle, path {
```

Comment:
If I'm not mistaken you can set `fill: none` right on the svg (e.g. on `.nx-binary-donut-chart`) and it will inherit down.

Reply:
> If I'm not mistaken you can set `fill: none` right on the svg (e.g. on `.nx-binary-donut-chart`) and it will inherit down.

Done e8460440bc68f3dd35da44cd427a523a3659fa99